testproject/Assets/Tests/Runtime/MultiprocessRuntime/Helpers/MultiprocessOrchestration.cs

```diff
@@ -66,6 +66,7 @@ public static void StartWorkerNode()
     // todo remove this once we have proper automation
     public static bool IsUsingUTR()
     {
-        return Environment.GetCommandLineArgs().Contains("-automated");
+        // return Environment.GetCommandLineArgs().Contains("-automated");
+        return false;
```

Comment:
Should we not remove the function and its usage, now ?

Reply:
I have a PR up that is removing this usage.